### PR TITLE
fix(task): try to upload log file even through the task has no log file

### PR DESCRIPTION
**Pull request by smallsheeeep**
#### What type of PR is this?
type-bug

#### What this PR does / why we need it:
This PR refine the code logic for uploading log files.

#### Which issue(s) this PR fixes:

Fixes #

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: server/odc-service/src/main/java/com/oceanbase/odc/service/flow/task/BaseODCFlowTaskDelegate.java [hunk 1]
```diff
@@ -336,6 +337,11 @@ public boolean cancel(boolean mayInterruptIfRunning) {
 
     private void setDownloadLogUrl(@NonNull Long flowInstanceId) throws IOException, NotFoundException {
         TaskEntity taskEntity = taskService.detail(taskId);
+        TaskType taskType = taskEntity.getTaskType();
```
Comment (yhilmare) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/flow/task/BaseODCFlowTaskDelegate.java:340]: we may use try-catch block to wrap this calling:
```java
File logFile = taskService.getLogFile(taskEntity.getCreatorId(), taskId + "", taskEntity.getTaskType(), OdcTaskLogLevel.ALL);
```

we may upload nothing when this calling throw any exceptions

Comment (smallsheeeep) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/flow/task/BaseODCFlowTaskDelegate.java:340]: The whole method `setDownloadLogUrl` is in a try-catch block.

Comment (smallsheeeep) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/flow/task/BaseODCFlowTaskDelegate.java]: Yes u are right, done!

**Review by yhilmare — APPROVED** [review]
lgtm

**Pull request merged**